test/TestProjects/Models-TypeSpec/src/Generated/ModelsTypeSpecModelFactory.cs

```diff
@@ -373,16 +391,6 @@ public static Int32ValuesFacet Int32ValuesFacet(string field = null, IEnumerable
             return new Int32ValuesFacet(field, serializedAdditionalRawData: null, values?.ToList(), value, kind);
         }
 
-        /// <summary> Initializes a new instance of <see cref="Models.DerivedWithDiscriminatorDefinedOnBase"/>. </summary>
-        /// <param name="optionalString"> Optional string. </param>
-        /// <param name="requiredString"> Required string. </param>
-        /// <param name="optionalInt"> Optional int. </param>
-        /// <returns> A new <see cref="Models.DerivedWithDiscriminatorDefinedOnBase"/> instance for mocking. </returns>
-        public static DerivedWithDiscriminatorDefinedOnBase DerivedWithDiscriminatorDefinedOnBase(string optionalString = null, string requiredString = null, int? optionalInt = null)
```

Comment:
This change is expected.
`DerivedWithDiscriminatorDefinedOnBase` has one required property that can be set via public constructor, and two optional properties that have public setters, so it doesn't need factory method to get instantiated.